addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15185,13 +15194,13 @@ msgstr ""
 #. Description of setting "Pictures -> Resolution" with label #169
 #: system/settings/settings.xml
 msgctxt "#36310"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #. Description of settings category "Pictures -> Slideshow" with label #108
 #: system/settings/settings.xml
 msgctxt "#36311"
-msgid "Category containing settings for how picture slideshows are handled."
+msgid "Category containing settings for how the picture slideshow is handled."
```

Comment:
I agree with Jalle19

Although only one slideshow is active at any one time, these settings will apply to any slideshow started at any point in time hence the pural. If anything the category "Slideshow" should be changed to "Slideshows" to fit in with other uses of pural, for example we use "Pictures" not "Picture" also in other areas we use "Videos" & "Movies" not "Video" & "Movie".